src/boot/bless-boot-generator.c

```diff
@@ -10,62 +9,47 @@
 #include "mkdir.h"
 #include "special.h"
 #include "string-util.h"
-#include "util.h"
 #include "virt.h"
 
-/* This generator pulls systemd-bless-boot.service into the initial transaction if the "LoaderBootCountPath" EFI
- * variable is set, i.e. the system boots up with boot counting in effect, which means we should mark the boot as
- * "good" if we manage to boot up far enough. */
+/* This generator pulls systemd-bless-boot.service into the initial transaction if the "LoaderBootCountPath"
+ * EFI variable is set, i.e. the system boots up with boot counting in effect, which means we should mark the
+ * boot as "good" if we manage to boot up far enough. */
 
-static const char *arg_dest = "/tmp";
```

Comment:
same here

Reply:
`DEFINE_MAIN_GENERATOR_FUNCTION()` always sets the destination. Hence the change is OK.

Reply:
Also added a comment in the commit message.